src/main/java/com/codepine/api/testrail/Request.java

```diff
@@ -129,9 +131,6 @@ public T execute() {
                     try (OutputStream outputStream = new BufferedOutputStream(con.getOutputStream())) {
                         JSON.writerWithView(this.getClass()).writeValue(outputStream, content);
                     }
-                } else {
-                    con.setFixedLengthStreamingMode(0);
-                    con.setDoOutput(true);
```

Comment:
My previous comment was only meant for line 134 i.e. con.setDoOutput is a common statement.